.gitignore

```diff
@@ -8,4 +8,4 @@ coverage-ts
 storybook-static
 .nyc_output
 *.tsbuildinfo
-cypress/
+/cypress/
```

Comment:
Without the leading `/`, we are ignoring any `cypress` anywhere in the repo... and our Cypress configuration lives in `config/cypress`. We don't want to ignore that dir!